ChopChop/ViewModels/OnlineRecipes/OnlineRecipeViewModel.swift

```diff
@@ -4,17 +4,24 @@ import Combine
 class OnlineRecipeViewModel: ObservableObject {
     private(set) var recipe: OnlineRecipe
 
+    private var creatorCancellable: AnyCancellable?
     private var recipeCancellable: AnyCancellable?
     private var followeesCancellable: AnyCancellable?
     private var firstRaterCancellable: AnyCancellable?
     private var imageCancellable: AnyCancellable?
     let storageManager = StorageManager()
 
+    @Published var creatorName = "No name" {
+        willSet { self.objectWillChange.send() }
+    }
+
     @Published private var firstRater = "No name"
     private var followeeIds: [String] = []
 
     @Published private(set) var image = UIImage(imageLiteralResourceName: "recipe")
```

Comment:
since this is gone, should we have a default image if theres no recipe? sth to symbolise no pic

Reply:
Sorry, I'm not sure what you mean by the image is gone, I think it's still used as the default image if there is no picture